tests/host_verify/host/host.cpp

```diff
@@ -193,7 +340,18 @@ int main()
         _verify_cert(CERT_RSA_BAD_FILENAME, false);
 
     if (_validate_file(REPORT_BAD_FILENAME, true))
-        _verify_report(REPORT_BAD_FILENAME, false);
+        _verify_report(REPORT_BAD_FILENAME, NULL, false);
+
+    //
+    // TODO: Report with collaterals tests.  Will to refactor the contentns
+    // of the collaterals to be self-contained in order to support
+    // serialization.
+    //
+    // if (_validate_file(REPORT_FILENAME, false))
+    //    _verify_report(REPORT_FILENAME, COLLATERALS_FILENAME, true);
+
+    // if (_validate_file(REPORT_BAD_FILENAME, true))
+    //     _verify_report(REPORT_FILENAME, COLLATERALS_BAD_FILENAME, false);
```

Comment:
THe code will be cleaner if unused code is removed.

Reply:
Done.